api/src/main/java/org/commonjava/maven/galley/model/Transfer.java

```diff
@@ -227,7 +227,7 @@ public boolean exists()
         OverriddenBooleanValue overriden = null;
         if ( decorator != null )
         {
-            overriden = decorator.decorateExists( this );
+            overriden = decorator.decorateExists( this, new EventMetadata(  ) );
```

Comment:
At some point we will probably need a variant of exists() that accepts an EventMetadata parameter, in case the existence decorator needs to know if a path has special properties within the scope of some package type.

Not critical now.